todoapp/projects/models.py

```diff
@@ -13,6 +18,16 @@ class Project(models.Model):
         Add string representation for this model with project name.
     """
 
+    name = models.CharField(max_length=100)
+    members = models.ManyToManyField(
+        AUTH_USER_MODEL,
+        through="ProjectMember",
+    ) 
+    max_members = models.PositiveIntegerField()
+    status = models.IntegerField(choices = Status.choices, default=Status.TO_BE_STARTED)
+
+    def __str__(self):
+        return f"Project {self.name}."
```

Comment:
This could be confusing for name to itself contain 'Project' in it
```suggestion
        return f"{self.name} - {self.status}"
```